src/states/node.rs

```diff
@@ -842,8 +842,8 @@ impl Node {
                             dst: Authority::NaeManager(prefix.lower_bound()),
                             content: MessageContent::GroupSplit(our_group_prefix),
                         };
-                        // We send on route 1 by default to avoid trying to send via the newly joined
-                        // node
+                        // We send on route 1 by default to avoid trying to
+                        // send via the newly joined node
```

Comment:
We have lots of comments that go beyond column 80, and rustfmt doesn't complain about them; I don't think we gain much by manually reformatting all of them.